https://patch-diff.githubusercontent.com/raw/openwall/john/pull/4822.patch

From dd9501341808fe19b9c5cd4f8f04edfc5f377077 Mon Sep 17 00:00:00 2001
From: Sam James <sam@gentoo.org>
Date: Wed, 22 Sep 2021 19:34:38 +0100
Subject: [PATCH] configure.ac: avoid bashism in pkg-config check

== is a Bashism and configure scripts need to work
with a POSIX shell, so let's change to = (like in
the other condition).

Signed-off-by: Sam James <sam@gentoo.org>
--- a/src/configure.ac
+++ b/src/configure.ac
@@ -347,7 +347,7 @@ AS_IF([test -z "$STRIP"], [AC_CHECK_TOOL([STRIP], [strip])])
 dnl Check if we have this at all
 PKG_PROG_PKG_CONFIG
 AC_MSG_CHECKING([if pkg-config will be used])
-if test "x$PKG_CONFIG" = x || test "x$enable_pkg_config" == xno ; then
+if test "x$PKG_CONFIG" = x || test "x$enable_pkg_config" = xno ; then
   JTR_MSG_RESULT_FAILIF_FORCED([xno], [x$enable_pkg_config], [pkg-config is NOT available])
   PKG_CONFIG=no
 else

